wolfcrypt/test/test.c

```diff
@@ -9438,6 +9438,7 @@ static wc_test_ret_t aes_xts_128_test(void)
         0x77, 0x8a, 0xe8, 0xb4, 0x3c, 0xb9, 0x8d, 0x5a
     };
 
+#ifndef HAVE_FIPS_VERSION /* FIPS requires different keys for main and tweak. */
```

Comment:
Should it be #ifndef HAVE_FIPS?

Reply:
fixed.  also fixed this in 8 other places.